lesson11/index.js

```diff
@@ -0,0 +1,84 @@
+const ul = document.getElementById('js-list');
+
+const createLoading = () => {
+    const li = document.createElement("li");
+    const loadingImg = document.createElement('img');
+
+    li.style.listStyle = "none";
+    li.id = "js-loading";
+    loadingImg.src = 'loading-circle.gif';
+    ul.appendChild(li).appendChild(loadingImg);
+};
+
+const removeLoading = () => document.getElementById('js-loading').remove();
+
+const createErrorMessage = (error) => {
+    const li = document.createElement("li");
+    li.textContent =`エラー内容：${error.message}`;
+    ul.appendChild(li);
+    console.error(error.message);
+};
+
+/**
+ * Async関数 
+ * @return {Array} 
+ */
+const jsonUrl = 'https://myjson.dit.upm.es/api/bins/4tpn';
+const fetchData = async () => {
+    try {
+        const response = await fetch(jsonUrl) // 中身はresponseオブジェクトを含むpromise
+        const json = await response.json(); //response オブジェクトから JSON を抽出
+        return json.data;
+
+    } catch(error) {
+        console.error(error);
+    } 
+};
+
+/**
+ * Async関数
+ * fetchData関数から値が返ってきた後、その値を返す
+ */
+const fetchListData = async () => {
+    createLoading();
+    try {
+        const data = await fetchData();
+        if( data.length === 0 ) {
+            throw new Error('データが空です。');
+        }
```

Comment:
IMO
This is what I learned from Morita-sensei.
If the data is empty, it is not an error.
Communication is successful without any problem.

I think we can just display the message without throwing an error.

Reply:
Thank you for review!
It only displays the message.
[cacda9f](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/11/commits/cacda9f2ea1a158d26abc71d60fb4e571ea75b05)